templates/admin/auth/new.tmpl

```diff
@@ -91,29 +91,29 @@
 			<h5 class="ui top attached header">{{ctx.Locale.Tr "admin.auths.tip.oauth2_provider"}}</h5>
 			<div class="ui attached segment">
 				<li>Bitbucket</li>
-				<span>{{ctx.Locale.Tr "admin.auths.tip.bitbucket"}}</span>
+				<span>{{ctx.Locale.Tr "admin.auths.tip.bitbucket" "https://bitbucket.org/account/user/<your username>/oauth-consumers/new"}}</span>
```

Comment:
`<your username>` triggers a false-positive orphan tag in djilint. Maybe workaround via `&lt;` and `&gt;`?

Reply:
… Then they'll be double encoded.

Reply:
Hmm how about `yourusername` or `{yourusername}`?

Reply:
I just used `[your username]`, but sure, `{…}` would work too

Reply:
`{…}` may be a bit more common 😆

Reply:
Done